rest/src/test/java/com/redhat/runtimes/inventory/web/TestLifecycleManager.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.web;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.smallrye.reactive.messaging.memory.InMemoryConnector;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+/**
+ * This class is based on TestLifecycleManager from RedHatInsights/notifications-backend:
+ * https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java
+ */
+public class TestLifecycleManager implements QuarkusTestResourceLifecycleManager {
+
+  PostgreSQLContainer<?> postgreSQLContainer;
```

Comment:
Should this field be private?

Reply:
Yeah looks like it could be made private instead